webroot/js/toolbar.js

```diff
@@ -94,9 +94,16 @@ if (elem) {
 
   // Bind on ready callback.
```

Comment:
I think we should update the comment here. The code has now a much higher implicit complexity than before.

Also the current comment only documents what we do,not why.
@D4rkMindz could you please rephrase it with the necessary details/background info?